defog/query.py

```diff
@@ -43,6 +43,13 @@ def execute_query_once(db_type: str, db_creds, query: str):
 
         cur.execute(query)
         colnames = [desc[0] for desc in cur.description]
+
+        # if there are any column names that are the same, we need to deduplicate them
```

Comment:
curious if we will need to drop the extra columns associated with the duplicate columns in the results?

Reply:
Ah no - because that column doesn't have duplicate _data_, it's just the same column name

For example, in the example below, the 2 columns are different, but will have the same column name when executed on Postgres!

```sql
SELECT a.revenue, b.revenue
FROM a
JOIN b ON a.product_id = b.product_id
```